### DISCUSSION TITLE
天才法厄同们，帮我算个积分，阿里嘎多。

### DISCUSSION THREAD

**Discussion by Lyfer123** [share121/inter-knot#3158] [其他]
$$\begin{aligned}
&\text { 设 } a \in \mathbb{R} \text {, 证明 }\\
&\lim _{t \rightarrow+\infty} e^{-t} t^{\frac{1}{2}-a} \int_1^{\infty} x^a\left(\frac{t e}{x}\right)^x \frac{d x}{x}=\sqrt{2\pi}
\end{aligned}$$

**Comment by Abandon12393** [comment]
=sqrt（2*pi）
